doc/hands_on/hands_on_dynamodb_vs.md

```diff
@@ -0,0 +1,229 @@
+# Hands on DynamoDB Virtual Schema
+
+In this Hands on guide we show you how to explore the new Virtual Schema for DynamoDB.
```

Comment:
Describe in a few sentences what is a DynamoDB and why people would need it. Also add a link to the DynamoDB documentation.